pom.xml

```diff
@@ -99,12 +99,12 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
-      <version>2.1.4</version>
+      <version>2.9.3</version>
```

Comment:
at this point we should modify the pom to include the bom, I can handle that if CI passes.

Reply:
using the BOM can come after the PR imo.

Reply:
done